Analysis/analysis.r

```diff
@@ -435,19 +516,22 @@ df_comparison <- rbind(df_1991 %>%
 df_comparison$bmi_class <- factor(df_comparison$bmi_class_final, levels = c("underweight", "normal", "overweight", "obese", "morbidly obese"))
 
 plotA <- df_comparison %>% 
-  filter(bmi_class %in% c("obese")) %>% 
+  filter(bmi_class %in% c("overweight", "obese", "morbidly obese")) %>% 
   ggplot(., aes(x = bmi_final, color = year)) + 
 geom_density(aes(weight = wt_int), size = 2, adjust = 2, alpha = 0.5) +
 facet_grid(bmi_class ~ . ) +
 theme_ipsum() +
-labs(title = "Obese: BMI Distribution",
+labs(title = "BMI Distribution",
      color = "",
      x = "BMI")+
   xlim(20,60) +
   theme_ipsum(base_size = 15, axis_title_size = 15, base_family="Averta") +
 scale_color_manual(values=c("#0000FF", "#F6A4B7")) +
   theme(legend.position = "top")
 
+ggsave(here("outputs/figures/bmi_difference_1991.png"), width = 10, bg='#ffffff')
```

Comment:
Looks like the title text in the right hand plot of this figure is cutt-off. I also think it might need to be saved in a higher resolution as its quite blurry for me.